migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator.rs

```diff
@@ -106,7 +107,7 @@ fn push_inline_relations(model: ModelWalker<'_>, table: &mut sql::Table) {
     for relation_field in relation_fields {
         let fk_columns: Vec<String> = relation_field.referencing_columns().map(String::from).collect();
 
-        // Optional unique index for 1:1 relations.
+        // Optional unique index for 1:1relations.
```

Comment:
I think it's a typo :p

Reply:
Yeah, first a cat and then I removed too much it seems ;-)